src/styles/main.scss

```diff
@@ -1,3 +1,21 @@
-body {
-  margin: 0;
+* {
+  box-sizing: border-box;
 }
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+}
+
+.content__catalog {
+  margin: 0 auto;
+}
```

Comment:
I decided to write in main.scss, because .content _catalog is the element of main. Overall I think this style must be in content.scss (file for semantic block main)